### ISSUE TITLE
Bug: latency

### ISSUE THREAD

**Issue by daiyi** [daiyitastic/random-wikipedia-newtab-extension#4]
Bug reported from a review: 

> Gets annoying when you are opening a new tab and after writing something in the URL the whole thing gets replaced by a Wikipedia page. So I have to wait for the Wikipedia URL to come before writing and that takes a second.

**Comment by jancborchardt** [comment]
Yup, same here. When anything is written in the bar, loading should probably just be canceled?
